superset-installer/etc/reflex-provisioner/inventory/templates/group_vars/global/all/raf/superset.yml

```diff
@@ -125,4 +125,13 @@ RVF_HELP_ENABLED: False
 SUPERSET_SESSION_LIFETIME_SECONDS: "3600" # stale session timeout
 RVF_LOG_LEVEL: "INFO"
 RVF_CACHE_DEFAULT_TIMEOUT: "{{ 60 * 60 * 24 }}"
-RVF_CACHE_TYPE: 'null'
\ No newline at end of file
+RVF_CACHE_TYPE: 'null'
+
+# Knox configuration
+RVF_IS_KNOX_SSO_ENABLED: "True"
```

Comment:
For other boolean variables, we are directly using `True` without string. Please check. Though it won't impact as inside the code, we are boolifying every variables with boolean.

Reply:
taken care for knoxsso related vars